source/extensions/filters/network/dubbo_proxy/router/router_impl.cc

```diff
@@ -331,7 +331,7 @@ void Router::UpstreamRequest::onPoolFailure(ConnectionPool::PoolFailureReason re
       reason == ConnectionPool::PoolFailureReason::RemoteConnectionFailure) {
     if (reason == ConnectionPool::PoolFailureReason::Timeout) {
       host->outlierDetector().putResult(Upstream::Outlier::Result::LocalOriginTimeout);
-    } else if (reason == ConnectionPool::PoolFailureReason::RemoteConnectionFailure) {
+    } else {
       host->outlierDetector().putResult(Upstream::Outlier::Result::LocalOriginConnectFailed);
     }
```

Comment:
Sorry for the delayed review. Could you add a simple test case for this minor change? Thanks.

Reply:
done